### PR TITLE
NN-3816 printed form view part 2

### PR DESCRIPTION
**Pull request by richardpopple**

### CODE REVIEW

File: server/routes/printReport/printReport.ts [hunk 1]
```diff
@@ -37,7 +41,21 @@ export default class PrintReportRoutes {
     const { user } = res.locals
     const { pdfMargins, adjudicationsUrl } = config.apis.gotenberg
     const adjudicationDetails = await this.reportedAdjudicationsService.getConfirmationDetails(adjudicationNumber, user)
-    const noticeOfBeingPlacedOnReportData = new NoticeOfBeingPlacedOnReportData(adjudicationNumber, adjudicationDetails)
+
+    const { reportedAdjudication, associatedPrisoner, prisoner } =
+      await this.decisionTreeService.reportedAdjudicationIncidentData(adjudicationNumber, user)
+    const offences = await this.decisionTreeService.getAdjudicationOffences(
```
Comment (sp-watson) [inline server/routes/printReport/printReport.ts:47]: A bit confused by this - the offences are available from the API call. is this for the questions and answers?

File: server/services/decisionTreeService.ts [hunk 2]
```diff
@@ -2,99 +2,115 @@ import { Decision } from '../offenceCodeDecisions/Decision'
...
+    return {
+      draftAdjudication,
+      incidentRole: data.incidentRole,
+      associatedPrisoner: data.associatedPrisoner,
+      prisoner: data.prisoner,
+    }
+  }
+
+  async reportedAdjudicationIncidentData(adjudicationNumber: number, user: User) {
+    const { reportedAdjudication } = await this.reportedAdjudicationService.getReportedAdjudicationDetails(
```
Comment (sp-watson) [inline server/services/decisionTreeService.ts:48]: Could do with confirmation of how this is working.

File: server/services/reportedAdjudicationsService.ts [hunk 3]
```diff
@@ -25,6 +29,10 @@ export default class ReportedAdjudicationsService {
     private readonly locationService: LocationService
   ) {}
 
+  async getReportedAdjudicationDetails(adjudicationNumber: number, user: User): Promise<ReportedAdjudicationResult> {
+    return new ManageAdjudicationsClient(user.token).getReportedAdjudication(adjudicationNumber)
```
Comment (sp-watson) [inline server/services/reportedAdjudicationsService.ts:33]: What's the difference between this and the below function?

File: server/routes/printReport/printReport.ts (hunk 1 above)
Comment (richardpopple) [reply server/routes/printReport/printReport.ts:47]: Yeah - all of this is to get the questions and answers (and fill them out with data - like prisoner names and whatnot)

File: server/services/reportedAdjudicationsService.ts (hunk 3 above)
Comment (richardpopple) [reply server/services/reportedAdjudicationsService.ts:33]: so the getConfirmationDetails pulls a bunch of stuff together - like neurodiversity etc. which we need for the non print page. And the getReportedAdjudication just brings back the adjudication - which we need for the actual printed page. I think we might be able to get away with not calling the getConfirmationDetails for the rendering of the pdf - but that will come in a later PR I think

**Review by sp-watson — APPROVED** [review]

**Pull request merged**